ws2812b/src/main/java/com/google/android/things/contrib/driver/ws2812b/ColorChannelSequence.java

```diff
@@ -0,0 +1,90 @@
+package com.google.android.things.contrib.driver.ws2812b;
+
+
+import android.support.annotation.ColorInt;
+import android.support.annotation.IntDef;
+import android.support.annotation.NonNull;
+
+import java.lang.annotation.Retention;
+import java.util.Arrays;
+
+import static java.lang.annotation.RetentionPolicy.SOURCE;
+
+class ColorChannelSequence {
+    @SuppressWarnings("WeakerAccess")
+    public static final int RGB = 1;
```

Comment:
instead of supressing the warning, you can change these to package access

Reply:
These constants are used as second parameter in the public accessible constructor of the Ws2812b class: 
```java
    public Ws2812b(String spiBusPort, @ColorChannelSequence.Sequence int colorChannelSequence) throws IOException {
        this (spiBusPort, new ColorToBitPatternConverter(colorChannelSequence));
    }
```
The failure was that the whole ColorChannelSequence class must be public accessible. Fixed: https://github.com/androidthings/contrib-drivers/pull/76/commits/37d29b8aa479e04f4f2bc49f85ee4d3872daa808